zppy/defaults/default.ini

```diff
@@ -138,10 +138,6 @@ ts_subsection = string(default="")
 [tc_analysis]
 # NOTE: always overrides value in [default]
 input_files = string(default="eam.h2")
-# TODO for v3.0.0: Remove this parameter
-# DEPRECATED. No longer used.
-# The scratch directory
-scratch = string(default="")
```

Comment:
I'm a little nervous to remove this parameter, before the test tc_analysis tests can pass on Perlmutter, let's keep your change now, and can decide latter if removing this parameter is okay, after Perlmutter change.

Reply:
This is just the default value for `scratch`. The actual use of `scratch` was removed in the `result_dir={{ scratch }}/tc-analysis_${Y1}_${Y2}/` -> `result_dir={{ output }}/post/atm/tc-analysis_${Y1}_${Y2}/` change (see https://github.com/E3SM-Project/zppy/pull/632/files#r1878679038).

Are you saying 1) undo that change or 2) just hold off on removing `scratch` as a parameter completely until we make sure that change works on Perlmutter?

Reply:
I'm wondering if we can test tc_analysis on Perl-mutter to make sure, it works without needing to including intermediate output under`scratch` file system? If so, we are free to remove `scratch` parameter and the codes what supports this.

Reply:
Yes, we can do this. I was holding off until the `rc` testing period, because I need to copy over all the test data, etc. to Perlmutter (and Compy). But I can try to do that earlier.

Reply:
This would be great, thank you!